civictechprojects/templates/new_index.html

```diff
@@ -1,26 +1,29 @@
 <!DOCTYPE html>
 <html lang="en" style="height: 100%">
-{% load django_seo_js %}
+  {% load django_seo_js %}
   <head>
     {% seo_js_head %}
-    <meta charset="utf-8">
+    <meta charset="utf-8" />
     <title>{{ title }}</title>
     <meta name="og:title" content="{{ title }}" />
     <meta name="description" content="{{ description }}" />
     <meta name="og:description" content="{{ description }}" />
     <meta name="og:type" content="{{ og_type }}" />
     <meta name="og:image" content="{{ og_image }}" />
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
     <script src="/static/js/runtime.bundle.js" defer></script>
     <script src="/static/js/main.bundle.js" defer></script>
     <script src="/static/js/vendors.bundle.js" defer></script>
-    <link rel="stylesheet" type="text/css" href="/static/css/main.styles.css"/>
-    <link rel="icon" type="image/png" href="{{FAVICON_PATH}}">
-    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css" integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU" crossorigin="anonymous">
-    {{ googleTagsHeadScript }}
-    {{ organizationSnippet }}
-    {{ hotjarScript }}
-    {{ googleScript }}
+    <link rel="stylesheet" type="text/css" href="/static/css/main.styles.css" />
+    <link rel="icon" type="image/png" href="{{FAVICON_PATH}}" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.5.0/css/all.css"
+      integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU"
+      crossorigin="anonymous"
+    />
+    {{ googleTagsHeadScript }} {{ organizationSnippet }} {{ hotjarScript }} {{
+    googleScript }}
```

Comment:
Seeing some problems with this section in Firefox and Chrome, where `{{ googleScript }}` is rendered literally. We'll likely want to set this block to be ignored or to not use Prettier on our HTML, depending.

![image](https://user-images.githubusercontent.com/16870466/105942926-0d616b00-6015-11eb-82cf-024efe54c837.png)